Makefile

```
@@ -30,15 +32,15 @@ container-dockerfile-dev:
 .PHONY: builder
 builder:
 ifdef BUILD_BUILDER_IMAGE
-	docker build \
+	docker buildx build --load --platform ${ARCH} \
```

Do we absolutely need `buildx` here? The legacy builder is deprecated and, AFAICT, docker now defaults to buildkit. Keeping it here seems harmless but might interfere with the workflow of people using `docker` symlinked to `podman`.
```suggestion
	docker build --load --platform ${ARCH} \
```

---

Okay. I just checked and it seems buildkit _is_ the default builder for docker. I can make this change. I assume the same needs to be done for the `image:`  target as well?